rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -246,6 +250,51 @@ def _decode_lending_pool_events(self, context: DecoderContext) -> DecodingOutput
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _maybe_enrich_liquidations(
+            self,
+            context: EnricherContext,
+    ) -> TransferEnrichmentOutput:
+        """
+        Process AAVE v2 liquidations. When a liquidation happens the user returns the debt token
+        and part of the collateral deposited is lost too. Those two events happen as transfers in
+        a transaction started by the liquidator.
+        """
+        for tx_log in context.all_logs:
+            if (
```

Comment:
What sense does it make?
```python
tx_log.topics[0] != LIQUIDATION_CALL or
tx_log.topics[0] != LIQUIDATION_CALL
```

Reply:
Also parenthesis won't hurt here to specify which condition (`or` or `and`) has higher priority